Order.java

```diff
@@ -0,0 +1,24 @@
+import java.util.Scanner;
+
+public class Order {
+  public static void main(String[] args) {
+    System.out.println("Write down any order of numbers separated by a space");
+    Scanner in = new Scanner(System.in);
+    String order = in.nextLine();
+    System.out.println("you have written: " + order);
+    String[] orderParts = order.split(" ");
+    int Length = orderParts.length;
+    try { 
+      for (int i = 0; i < Length - 1; i++) {
```

Comment:
do not use tmp variable, use orderParts.length instead Length. java compiler is smart enough to optimize such things